webapp/components/logged_in.jsx

```diff
@@ -176,6 +184,14 @@ export default class LoggedIn extends React.Component {
             user: this.state.user
         });
     }
+
+    onFocusListener() {
+        BrowserStore.setGlobalItem('app-focus', true);
```

Comment:
Could you actually move these lines that set the `BrowserStore` into an "action" function or two in `global_actions.jsx` so that we aren't talking to the browser store directly? It's a bit more complicated, but theoretically, only the stores themselves should be talking to the `BrowserStore` and updating it only in response to those actions. It'll also help should we ever have any other code that relies on if the window is focused.

If you're not familiar with that code, you'll need to add a function or two to `global_actions.jsx` similar to the `show*Modal` functions in that class that will just dispatch a view action when the window is focused or blurred. You can then listen for those events in the `NotificationStore.dispatchToken` function and use that to either set a variable or something in the `BrowserStore` to store if the window is focused.